src/app/newSettings/Sections/Workspace/Teams/components/TeamsList.tsx

```diff
@@ -0,0 +1,54 @@
+import { t } from 'i18next';
+import { WorkspaceTeamResponse } from '@internxt/sdk/dist/workspaces/types';
+
+import Button from 'app/shared/components/Button/Button';
+
+interface TeamsListProps {
+  setCreateTeamDialogOpen: (value: boolean) => void;
+  teams: WorkspaceTeamResponse;
+  isCurrentUserWorkspaceOwner: boolean;
+}
+
+const TeamsList: React.FC<TeamsListProps> = ({ setCreateTeamDialogOpen, teams, isCurrentUserWorkspaceOwner }) => {
+  return (
+    <>
+      <div className="mt-2 flex items-center justify-between">
+        <h2 className="text-xl font-medium text-gray-100">
+          <span>{teams.length} </span>
+          <span>
+            {teams.length != 1 ? t('preferences.workspace.teams.title') : t('preferences.workspace.teams.team')}
+          </span>
+        </h2>
+        {isCurrentUserWorkspaceOwner && (
+          <Button variant="primary" onClick={() => setCreateTeamDialogOpen(true)}>
+            {t('preferences.workspace.teams.createTeam')}
+          </Button>
+        )}
+      </div>
+      <div className="rounded-xl">
+        <div className="grid h-12 grid-cols-3 rounded-t-xl border border-gray-10 bg-gray-1 py-2 text-base font-medium">
+          <div className="col-span-2 flex items-center border-r border-gray-10 pl-5">
+            {t('preferences.workspace.teams.team')}
+          </div>
+          <div className="col-span-1 flex items-center pl-5">{t('preferences.workspace.teams.members')}</div>
+        </div>
+        {teams.map((team) => {
+          const isLastTeam = teams.length === teams.indexOf(team) + 1;
+          return (
+            <div
+              key={team.team.id}
+              className={`grid h-12 cursor-pointer grid-cols-3 border-x border-b border-gray-10 bg-surface py-2 text-base font-medium hover:bg-gray-5 ${
+                isLastTeam && 'rounded-b-xl'
+              }`}
+            >
+              <div className="col-span-2 flex items-center pl-5">{team.team.name}</div>
```

Comment:
The default team name arrives `null`, is this a known problem?
We should check with someone in the backend to make sure it has a default name.
I also think something like a hyphen should be added in the case of no name so that it doesn't give the impression that it is not being displayed.

Reply:
I have already left it in a comment for backend, I think it should not be in this list since it is really the wokspace.
I keep it in consideration to solve it.